packages/angular-cli/tasks/e2e.ts

```diff
@@ -1,15 +1,21 @@
 const Task = require('../ember-cli/lib/models/task');
 import * as chalk from 'chalk';
-import {exec} from 'child_process';
-
+import { exec } from 'child_process';
+import { E2eOptions } from '../commands/e2e';
 
 export const E2eTask = Task.extend({
-  run: function () {
+  run: function (options: E2eOptions) {
+    const commandArgs: string[] = [];
     const ui = this.ui;
     let exitCode = 0;
 
+    if (options.suite) {
+      commandArgs.push(`--suite=${options.suite}`);
```

Comment:
You're feeding in concatenated strings from the input, which would remove quotes etc. Could you surround the argument with double quotes?